### PR TITLE
Warrior dev+imx6ul des0258 mbl

### PR DESCRIPTION
**Pull request by bryanodonoghue**
Add imx6ul des0258 as a board type to conf.

Small change is required to enable : https://github.com/ARMmbed/meta-mbl/pull/599

### CODE REVIEW

**Comment by diego-sueiro** [comment]
Sync: https://github.com/ARMmbed/meta-mbl/pull/599

File: bblayers_imx6ul-des0258-mbl.conf [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# LAYER_CONF_VERSION is increased each time build/conf/bblayers.conf
+# changes incompatibly
+
+# These layers hold machine specific content, aka Board Support Packages
+BSPLAYERS += " \
+  ${OEROOT}/layers/meta-mbl/meta-freescale-mbl \
+  ${OEROOT}/layers/meta-freescale \
+  ${OEROOT}/layers/meta-mbl/meta-freescale-3rdparty-mbl \
+  ${OEROOT}/layers/meta-freescale-3rdparty \
+  ${OEROOT}/layers/meta-mbl/meta-fsl-bsp-release-mbl/imx/meta-bsp \
```
Comment (diego-sueiro) [inline bblayers_imx6ul-des0258-mbl.conf]: `meta-fsl-bsp-release-mbl` and `meta-fsl-bsp-release` layers are not needed. This platform doesn't have the qca9377 wireless module.

File: setup-environment-internal [hunk 2]
```diff
@@ -72,7 +72,7 @@ mbl_eula_handle () {
         # EULA. If we start using recipes covered by a EULA without updating this
         # code then those recipes will fail to build due to the missing
         # ACCEPT_FSL_EULA variable.
-    imx8mmevk|imx7d-pico)
+    imx8mmevk|imx7d-pico|imx6ul-pico|imx6ul-des0258)
```
Comment (diego-sueiro) [inline setup-environment-internal]: EULA is not needed for `imx6ul-des0258`.

Comment (diego-sueiro) [reply setup-environment-internal]: And `imx6ul-pico`  will be added by PR #52 .

**Comment by bryanodonoghue** [comment]
So, thanks for pointing that out.
All of your feedback ought to be addressed now.

**Comment by jjohnson-arm** [comment]
Please don't delete the branch....

**Pull request merged**